src/main/java/io/jenkins/plugins/audit/config/AuditLogRootAction.java

```diff
@@ -0,0 +1,32 @@
+package io.jenkins.plugins.audit.config;
+
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.model.DirectoryBrowserSupport;
+import hudson.model.RootAction;
+import jenkins.model.Jenkins;
+
+
+@Extension
+public class AuditLogRootAction implements RootAction {
+    @Override
+    public String getIconFileName() {
+        return "notepad.png";
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Audit Logs";
+    }
+
+    @Override
+    public String getUrlName() {
+        return "auditLog" ;
+    }
+
+    public DirectoryBrowserSupport doView() {
```

Comment:
```suggestion
    public DirectoryBrowserSupport doIndex() {
```

If you name this `index`, it will be available as the index page. This makes the root action link directly to the browser. If we create a fancier dashboard page, we can always move the directory browser to a different URL.